.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,91 @@
+name: Bug Report
+description: File a bug report to help us improve.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Description
+        Thanks for taking the time to fill out this bug report to help us improve Noir!
+  - type: textarea
+    id: aim
+    attributes:
+      label: Aim
+      description: Describe what you tried to achieve.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: Describe what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: bug
+    attributes:
+      label: Bug
+      description: Describe the bug. Supplement error codes / terminal logs if applicable.
```

```suggestion
      description: Describe the bug. Supply error codes / terminal logs if applicable.
```

This makes this description consistent with the global template.